lib/card_generator.rb

```diff
@@ -0,0 +1,23 @@
+require './lib/card'
+
+class CardGenerator
+  attr_reader :cards, :filename
+  def initialize(filename)
+    @filename = filename
+    @cards = cards
```

Comment:
Where is the variable `cards` coming from? Would it have been better to start the instance variable `@cards` off as an empty array and shovel (`<<`) cards into it as they were being created in your `sort_file_into_deck` method? This would eliminate the need for the `card_collector` variable as well.